package.json

```diff
@@ -3,17 +3,33 @@
   "version": "3.1.0",
   "repository": "git://github.com/segmentio/snippet.git",
   "description": "Templating methods for rendering the analytics.js snippet.",
+  "main": "lib/index.js",
+  "dependencies": {
+    "json3": "^3.3.2"
```

Comment:
If you rely on JSON3 here you'll need to wrap the entire bundle in a shim that prevents it from messing with any AMD loader on the page.